### DISCUSSION TITLE
Choix des catégories

### DISCUSSION THREAD

**Discussion by WolfyzDBois** [WolfyzDBois/Ressources#20] [General]
Je lance une discussion ici : quel devrait être le découpage des sections selon vous ? 
(pour trier les ressources dans des dossiers; une ressource ne doit évidemment être que dans une catégorie) 
@Sn0wAlice @ena0 @WolfyzDev

**Comment by ena0** [comment]
pour moi il faudrait faire un découpage suivant les os compatible,
suivant la catégorie de la ressource (logiciel, site internet, image, etc...)
et suivant ce à quoi elle sert (web design, JavaScript, IA, etc..)

**Reply by WolfyzDBois** [reply]
Tu verrais trois découpage différent ou un comme ceci ? 
| Windows 
| -- | Logiciel 
| Linux 
| -- | Logiciel 

Ou autre type ?

**Reply by ena0** [reply]
je pensait plus avoir des "label" en lien avec les trois catégories que j'ai mis avec un moyen de filtrer

**Reply by WolfyzDBois** [reply]
Oui comme sur GitHub ?

**Comment by WolfyzDBois** [comment]
@Sn0wAlice 
Tu penses que ce serait possible de créer des sortes de label comme sur GitHub sur notre site ? (pas forcément exactement comme ca mais un systeme du meme genre ? )
Ce serait l'idéal pour trier les ressources, comme ceci : 
![exemple label](https://user-images.githubusercontent.com/93478889/174602827-19181b66-4d2b-47eb-9808-4df9a5e2a288.png)

**Reply by Sn0wAlice** [reply]
Look la doc https://retype.com/components/badge/

**Reply by WolfyzDBois** [reply]
me sens débile, soudainement 🤣

**Comment by WolfyzDBois** [comment]
Ok ce que je propose c'est qu'on les range par catégorie du type : Front-End / Back-end / C++ / Python / Design UI/UX, etc ... 
Et que pour chacunes d'entre elles on rajoute des labels : 
- Un pour le type (site, fichier ); si logiciel : quelles os compatibles 
- le sujet plus précis 
etc ... 

Bien sur on fera un sommaire.

**Reply by WolfyzDBois** [reply]
@Sn0wAlice @ena0 @julescrepi @LACOSTAR91 Vous en pensez quoi ? (et vous feriez quoi comme grande catégorie ? )

**Reply by Sn0wAlice** [reply]
je valide
